doc/modules/mod_event_pusher_rabbit.md

```diff
@@ -128,6 +128,11 @@ as well. Provided metrics:
   * `message_payload_size` - size of a messages (in bytes) that was published to
   a RabbitMQ server
 
+> All the above metrics have a prefix which looks as follows:  
+> `mongooseim.<xmpp_host>.backends.mod_event_pusher_rabbit.<metric_name>`.
+> For example a proper metric name would look like:
+> `mongooseim.localhost.backends.mod_event_pusher_rabbit.rabbit_connections_active`
```

Comment:
Then, the `rabbit` in the last component of the metric name is redundant. I think it's worth getting rid of it as the name is already a quite long one.

Reply:
@mentels metrics names fixed. It required another PR (#1928).